runtime/compiler/env/j9jit.tdf

```diff
@@ -90,26 +90,41 @@ TraceEvent=Trc_JITServerMethodSuccessfullyLoaded Overhead=1 Level=2 Test Templat
+TraceEvent=Trc_JITServerClearedSessionCaches Overhead=1 Level=1 Test Template="JITServer: compThreadID=%d clientSessionData=%p clientUID=%llu (seqNo=%u, expectedSeqNo=%u, numActiveThreads=%d) has cleared the session cache detachedEntry=%p. Setting expectedSeqNo from %u to %u"
+TraceEvent=Trc_JITServerThreadGoSleep        Overhead=1 Level=1 Test Template="JITServer: compThreadID=%d clientSessionData=%p clientUID=%llu (seqNo=%u, expectedSeqNo=%u, numActiveThreads=%d) which previously timed-out will go to sleep again. Possible reasons waitToBeNotified=%d"
+TraceEvent=Trc_JITServerUpdateSeqNo          Overhead=1 Level=1 Test Template="JITServer: compThreadID=%d clientSessionData=%p clientUID=%llu (seqNo=%u, expectedSeqNo=%u, numActiveThreads=%d) updating expectedSeqNo from %u to %u"
+TraceEvent=Trc_JITServerOutOfSequenceMessage Overhead=1 Level=1 Test Template="JITServer: compThreadID=%d clientSessionData=%p clientUID=%llu (entry=%p) (seqNo=%u, expectedSeqNo=%u, numActiveThreads=%d) out-of-sequence msg detected. Parking this thread"
+TraceEvent=Trc_JITServerDiscardMessage       Overhead=1 Level=1 Test Template="JITServer: compThreadID=%d clientSessionData=%p clientUID=%llu (seqNo=%u, expectedSeqNo=%u, numActiveThreads=%d) discarding the older msg"
+TraceEvent=Trc_JITServerUnexpectedSeqNo      Overhead=1 Level=1 Test Template="JITServer: compThreadID=%d clientSessionData=%p clientUID=%llu (seqNo=%u, expectedSeqNo=%u, numActiveThreads=%d) unexpected seqNo"
+
+TraceEvent=Trc_JITServerInitCHTable          Overhead=1 Level=1 Test Template="JITServer: compThreadID=%d clientSessionData=%p clientUID=%llu initialize CHTable size %llu"
+TraceEvent=Trc_JITServerAbortInitCHTable     Overhead=1 Level=1 Test Template="JITServer: compThreadID=%d clientSessionData=%p clientUID=%llu CHTable is not empty size %llu. Abort the update size %llu"
+TraceEvent=Trc_JITServerDoCHTableUpdate      Overhead=1 Level=1 Test Template="JITServer: compThreadID=%d clientSessionData=%p clientUID=%llu do CHTable update modified %llu and removed %llu"
+
+TraceEvent=Trc_JITServerUnloadClasses        Overhead=1 Level=1 Test Template="JITServer: compThreadID=%d clientSessionData=%p clientUID=%llu will process a list of %llu unloaded classes"
+TraceEvent=Trc_JITServerFreeCHTable          Overhead=1 Level=1 Test Template="JITServer: compThreadID=%d clientSessionData=%p clientUID=%llu will free CHTable"
 
 TraceEvent=Trc_JITServerStreamFailure                Overhead=1 Level=1 Test Template="JITServer: compThreadID=%d (%s) stream failure (compiling %s @ %s): %s"
 TraceEvent=Trc_JITServerStreamVersionIncompatible    Overhead=1 Level=1 Test Template="JITServer: compThreadID=%d (%s) stream version incompatible (compiling %s @ %s): %s"
 TraceEvent=Trc_JITServerStreamMessageTypeMismatch    Overhead=1 Level=1 Test Template="JITServer: compThreadID=%d (%s) stream message type mismatch (compiling %s @ %s): %s"
```

Comment:
Not sure if this is true any longer, or that we care about this for JITServer but I believe the trace point IDs of existing tracepoints will change if you add new tracepoints in the middle of one of these `.tdf` files. Probably doesn't matter here as I doubt anything is consuming these.

Reply:
I think the trace point IDs of existing tracepoints will change in this case. I didn't find the existing code that consumes these IDs. So far all the new JITServer tracepoints have been added at the end of ` j9jit.tdf` which wouldn't affect the existing Non-JITServer tracepoint IDs. We're still rearranging the JITServer related tracepoints. Thanks for pointing this out. It's very good to keep it in mind when making the changes later on.